Alert/README.md

```diff
@@ -0,0 +1,32 @@
+# Alert data model
+
+## Description
+This entity models an alert generated either by a user or by a device. This models could be used to send alerts related to traffic jam, accidents, weather conditions, high level of pollutants and so on.
```

Comment:
I would not say anything about who generates the alert, it could be an automated system, a CEP for instance, checking whether certain conditions are met.

Reply:
My suggestion here is to have an attribute named `alertSource` which can be a reference to a Device, a person, system, etc.

Reply:
would be good enough source ? if I recall correctly it's defined in GSMA commons, even if just as string and not a referentiable uri

Reply:
that's a different thing. `source` is the original data source, i.e. a data source that probably is not even harmonised and which was used to produce this other data source